src/main/java/com/ft/methodearticlemapper/transformation/StructuredMethodeSourcedBodyXMLEventHandlerRegistryInnerTable.java

```diff
@@ -28,12 +28,15 @@ public StructuredMethodeSourcedBodyXMLEventHandlerRegistryInnerTable(MethodeBody
 		this.registerStartAndEndElementEventHandlerWrapper(new StripElementAndContentsXMLEventHandler(), "videoPlayer", "web-pull-quote", "plainHtml", "web-background-news", "promo-box");
 
 		this.registerStartAndEndElementEventHandlerWrapper(new RetainWithSpecificAttributesXMLEventHandler("colspan", "rowspan"),
-				"th", "td");
+				"td");
 
 		this.registerStartAndEndElementEventHandlerWrapper(new RetainWithoutAttributesXMLEventHandler(),
 				"thead", "tbody",
 				"tfoot", "caption", "tr");
 
+		this.registerStartAndEndElementEventHandlerWrapper(new RetainWithSpecificAttributesXMLEventHandler("data-coloumn-name", "data-column-type", "data-column-sortable", "data-column-default-sort", "data-column-hidden", "data-column-width", "data-column-filter-type", "colspan", "rowspan"),
+				"th");
```

Comment:
A `th` tag can also contain the attibutes: `id`, `headers`, `scope`, `data-name`.

Reply:
You're not handling the `col` and `colgroup` tags.

Reply:
You're not handling `colgroup` and `col` tags.